aldryn_forms/templates/admin/aldryn_forms/display/submission_data.html

```diff
@@ -0,0 +1,7 @@
+{% load i18n %}
+{% filter linebreaksbr %}{% spaceless %}{% for field in data %}
+{% trans "Field" %}: {{ field.name }}
+{% trans "Label" %}: {{ field.label }}
+{% trans "Value" %}: {{ field.value }}
+{% endfor %}{% endspaceless %}
+{% endfilter %}
```

Comment:
@czpython I think there this is due to the email generation, shouldn't harm to add last line or?